datetime/mod.ts

```diff
@@ -22,14 +27,22 @@ export function parseDate(dateStr: string, format: string): Date {
   return new Date(Number(y), Number(m) - 1, Number(d));
 }
 
+export type DateTimeFormat = 
+  "mm-dd-yyyy hh:mm"
+  | "dd-mm-yyyy hh:mm" 
+  | "yyyy-mm-dd hh:mm"
+  | "hh:mm mm-dd-yyyy"
+  | "hh:mm dd-mm-yyyy"
+  | "hh:mm yyyy-mm-dd"
```

Comment:
missing a semi-colon here.. be sure to run format.ts

Reply:
@ry thanks! I updated it.